web/src/features/projects/server/projectsRouter.ts

```diff
@@ -105,6 +111,9 @@ export const projectsRouter = createTRPCRouter({
         action: "delete",
       });
 
+      // TODO: Set deleted_at to current timestamp instead of deleting the project
+      // Afterwards, send message to ProjectDelete queue for asynchronous cleanup.
+      // Do we want to remove all corresponding trace information or does that happen automatically using Prisma cascade?
```

Comment:
Validate assumptions here: Do we actually cascade via Prisma or how do we ensure that everything is deleted?

Further TODO: Delete things from ClickHouse as well.